modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -9,13 +9,17 @@ In the table, features are marked with the following statuses:
 TP:: Technology Preview
 GA:: General Availability
 
+// Writer, see https://spaces.redhat.com/pages/viewpage.action?spaceKey=SRVKP&title=Pipeline+Component+Matrix
+
 .Compatibility and support matrix
 [options="header"]
 |===
 
 | {pipelines-title} Version 7+| Component Version | OpenShift Version | Support Status
 
-| Operator | Pipelines | Triggers | CLI | Catalog | Chains | Hub | PAC | | 
+| Operator | Pipelines | Triggers | CLI | Catalog | Chains | Hub | PAC | |
+
+// |1.8 | 0.37.x | 0.20.x | 0.24.x | NA | 0.9.0 (TP) | 1.8.x (TP) | 0.10.x (TP) | 4.10, 4.11, 4.12 (planned) | GA
```

Comment:
This line is intentionally commented out. @sounix000 will uncomment it when Pipelines 1.8 goes live (after OCP 4.11 GA).